app/controllers/admin/api/buyers_applications_controller.rb

```diff
@@ -282,11 +282,12 @@ def application_plan
   end
 
   def application_params
-    flat_params.slice(*application_attributes)
+    flat_params.permit(application_attributes | %w[user_key application_id plan_id])
```

Comment:
Seems like `plan_id` was not needed in the past. Would you explain why add it now?